students/surowiec_marek/lesson_03_funcions/the_collatz_sequence.py

```diff
@@ -0,0 +1,13 @@
+# The Collatz Sequence
+def collatz(number):
+    if number % 2 == 0:
+        return number // 2
+    elif number % 2 == 1:
+        return 3 * number + 1
```

Comment:
Zgodnie z poleceniem:

> If number is even, then collatz() should print number // 2 and return this value

Czyli to funkcja ma wyświetlać oraz zwracać. U Ciebie funkcja zwraca, a wyświetlanie jest w zewnętrznym loopie. Zalecam poprawkę, żeby było zgodnie z poleceniem.